doc/source/management/ssl-improvement.rst

```diff
@@ -0,0 +1,100 @@
+.. _percona-server.management.ssl:
+
+================================================================================
+SSL Improvements
+================================================================================
+
+|Percona Server| enables transmitting data in the encrypted form by using the
+TLSv1.2 protocol. By default, |Percona Server| disables TLSv1.0 and deprecates
+TLSv1.1.
+
+.. rubric:: Implemented Using |openssl|
+
+|Percona Server| uses the |openssl| toolkit version 1.1 to implement the suport
+of TLSv1.2 protocol. As part of its implementation, |Percona Server| offers
+correct diagnostic messages.
```

Comment:
Still too much tied to openssl

Reply:
ready for review

Reply:
Still too much tied to openssl and/or to TLSv1.2

Reply:
version numbers removed. if this is not enough you have to define what you mean by `still too much tied`

Reply:
The fixed diagnostics are not related to 1) OpenSSL specifically 2) TLS specifically 3) YaSSL specifically. And this paragraph mixes OpenSSL, TLS, and fixed diagnostics: OpenSSL was not linked with in order to support newer TLS versions, please separate this; the fixed diagnostics are not a consequence of TLS nor OpenSSL support work, and is orthogonal to the SSL library in use